lib/jekyll-admin/server/collection.rb

```diff
@@ -47,26 +49,70 @@ def collection
         collection[1] if collection
       end
 
+      def request_path
+        sanitized_path File.join(site.source, request_payload["path"])
```

Comment:
Oh, ok 👍  
The only problem is that this adds extra `/` at the end. Is there a convenient way to delete it?